### ISSUE TITLE
Throwing XPathException not reported

### ISSUE THREAD

**Issue by dizzzz** [eXist-db/eXide#41]
When a java-function throws a XPathException, it is not visible in the UI;
only the log file might give a hint.....

**Comment by dizzzz** [comment]
work around:

```
    } catch (XPathException ex) {
        LOG.error(ex.getMessage());
        ex.setLocation(this.line, this.column, this.getSource());
        throw ex;
    }
```

**Comment by wolfgangmm** [comment]
Should be fixed by ef1682038c3d9ee658a07d0cef8caa4dd0013d69
